src/platform-implementation-js/dom-driver/gmail/widgets/buttons/button-view.ts

```diff
@@ -41,11 +43,12 @@ export default class ButtonView implements ButtonViewI {
   private _eventStream: Bus<any, any>;
 
   constructor(options: ButtonViewOptions) {
+    const { iconLigaDefault, iconClass } = options;
     this._hasDropdown = false;
     this._isEnabled = options.enabled !== false;
 
-    this._iconClass = options.iconClass;
-    this._iconUrl = options.iconUrl;
```

Comment:
why is this line removed, should it be there in addition not instead?

Reply:
Good catch. See [3f42061](https://github.com/InboxSDK/InboxSDK/commit/3f420617fcc7285be1bdbb93c51de7c6841cacec)